dashboard/test/ui/features/teacher_tools/rubrics/teacher_view_of_rubric.feature

```diff
@@ -17,16 +17,19 @@ Scenario: Teachers can give and send feedback on the rubric to students.
 
   # Teacher can see and submit feedback for a student
   Then I sign in as "Teacher_Lillian" and go home
+  And I add the current user to the "ai-rubrics" single section experiment for the "allthethings" course
   And I am on "http://studio.code.org/s/allthethings/lessons/48/levels/2?enableExperiments=ai-rubrics"
   And I wait to see "#ui-floatingActionButton"
   And I wait until element "#teacher-panel-container" is visible
   And I wait until element ".student-table" is visible
   And I click selector ".student-table tr:nth(1)" to load a new page
   And I click selector "#ui-floatingActionButton" once I see it
-  And I wait until element "h6:contains(Code Quality)" is visible
+  And I wait until element "h5:contains(Code Quality)" is visible
   And I wait until element "button:contains(Extensive)" is visible
   Then I click selector "button:contains(Extensive)"
   And I click selector "#ui-teacherFeedback" once I see it
+
+  When I wait for 3 seconds
```

Comment:
an explicit wait like this is a sign of future flakiness. is there anything in the UI where you can wait for it to change or become visible instead?

Reply:
I think I can just delete that line... it is racing the update to the textarea from opening the modal with the old data... so the problem might have been that you type in the thing but it gets replaced with empty string.

To be entirely certain, it's probably good to disable the textarea until it negotiates prior feedback at least once.

Reply:
disabling the textarea while the data loads makes sense to me! https://codedotorg.atlassian.net/browse/AITT-498

Reply:
Wasn't very hard. Could pull out into a separate PR if you'd like.

Reply:
no that's fine, I was just hedging my bets in case you didn't have time to get to it this week. I can close out that issue when this PR merges